specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/scheduledEvents.json

```diff
@@ -0,0 +1,175 @@
+              "$ref": "./managedClusters.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Scheduled Event": {
+            "$ref": "./examples/ScheduledEventsGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScheduledEvent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of a scheduled event."
+        },
+        "eventId": {
+          "type": "string",
+          "description": "The event id of a scheduled event."
+        },
+        "eventSource": {
+          "type": "string",
+          "description": "The source of a scheduled event."
+        },
+        "eventStatus": {
+          "type": "string",
+          "enum": [
+            "Scheduled",
+            "Started",
+            "Completed",
+            "Cancelled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "eventStatus",
+            "modelAsString": true
+          },
+          "description": "The status of a scheduled event."
+        },
+        "scheduledTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event starts."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event last updates."
+        },
+        "resources": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "arm-id",
+            "x-ms-arm-id-details": {
+              "allowedResources": [
+                {
+                  "type": "Microsoft.ContainerService/managedClusters"
+                },
+                {
+                  "type": "Microsoft.ContainerService/managedClusters/agentPools"
+                }
+              ]
+            }
+          },
+          "description": "The list of resources of a scheduled event."
+        },
+        "resourceType": {
+          "type": "string",
+          "enum": [
+            "ManagedCluster",
+            "AgentPool"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceType",
+            "modelAsString": true
+          },
+          "description": "The resource type of a scheduled event."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ScheduledEventNameParameter": {
+      "name": "scheduledEventName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 63,
+      "pattern": "^[a-zA-Z0-9]$|^[a-zA-Z0-9][-_a-zA-Z0-9]{0,61}[a-zA-Z0-9]$",
+      "description": "The name of the scheduled event.",
+      "x-ms-parameter-location": "method"
```

Comment:
This `x-ms-*` annotation is invalid for this parameter:

> This extension can only be applied on global parameters. If this is applied on any parameter in an operation then it will be ignored.

ref: https://azure.github.io/autorest/extensions/#x-ms-parameter-location

Reply:
Removed.